DigixGlobal/governance-ui-components
# [Feature] - Add support for uploading and displaying PDF in project details

Ref [DGDG-509](https://tracker.digixdev.com/agiles/88-14/89-22?issue=DGDG-509)

This allows users to upload images and PDF files when create/editing Projects.

Note:  There's still a need to update the css so PDF files are displayed properly

Test Plan
 - Create a project, uploading images and PDF (both single and multiple pages) and check if pdf files are showing
 - Edit a project, upload/remove files and see if files are  removed/shown as expected
 - View created/edited projects and verify that PDF files are also showing on the details
